### PR TITLE
Update readme with description to run descheduler as a job inside a pod.

### PR DESCRIPTION
**Pull request by aveshagarwal**
@ravisantoshgudimetla in case if you have any feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,109 @@ For more information about available options run:
 $ ./_output/bin/descheduler --help
 ```
 
+## Running Descheduler as a Job Inside of a Pod
+
+Descheduler can be run as a job inside of a pod. It has the advantage of
+being able to be run multiple times without needing user intervention.
+Descheduler pod is run as a critical pod to avoid being evicted by itself,
+or by kubelet due to an eviction event. Since critical pods are created in
+`kube-system` namespace, Descheduler job and its pod will also be created
```
Comment (ravisantoshgudimetla) [inline README.md]: s/Descheduler/descheduler

File: README.md [hunk 2]
```diff
@@ -40,6 +40,109 @@ For more information about available options run:
 $ ./_output/bin/descheduler --help
 ```
 
+## Running Descheduler as a Job Inside of a Pod
```
Comment (ravisantoshgudimetla) [inline README.md]: Can we change the name to incluster configuration for descheduler?

File: README.md [hunk 3]
```diff
@@ -40,6 +40,109 @@ For more information about available options run:
...
+
+First we create a simple Docker image utilizing the Dockerfile found in the root directory:
+
+```
+$ make image
+```
+
+### Create a cluster role
+
+To give necessary permissions to the descheduler pod, create a cluster role:
```
Comment (ravisantoshgudimetla) [inline README.md]: s/descheduler pod/for descheduler to work in a pod/ ?

File: README.md (hunk 2 above)
Comment (aveshagarwal) [reply README.md]: i think many users might not be familar with the term incluster configuration.

**Pull request merged**